lib/workers/global/index.js

```diff
@@ -56,6 +56,15 @@ async function start() {
       delete config[key];
     });
     global.trustLevel = config.trustLevel || 'low';
+    if (global.trustLevel === 'low') {
+      global.env = {
```

Comment:
Instead of global, let's create a new file lib/util/env.js that exports a function `getUntrustedEnv()`.